Source/SpatialGDK/Public/Interop/SpatialReceiver.h

```diff
@@ -72,6 +72,20 @@ struct FObjectReferences
 	UProperty*							Property;
 };
 
+struct FPendingIncomingRPC
+{
+	FPendingIncomingRPC(const TSet<UnrealObjectRef>& InUnresolvedRefs, UObject* InTargetObject, UFunction* InFunction, const TArray<uint8>& InPayloadData, int64 InCountBits)
+		: UnresolvedRefs(InUnresolvedRefs), TargetObject(InTargetObject), Function(InFunction), PayloadData(InPayloadData), CountBits(InCountBits) {}
+
+	TSet<UnrealObjectRef> UnresolvedRefs;
+	TWeakObjectPtr<UObject> TargetObject;
+	UFunction* Function;
```

Comment:
What if this was pointing to a Blueprint function that got unloaded? TWeakObjectPtr<>?

Reply:
The function can't get unloaded unless the target object does, in which case it won't try to call the function anyway.